src/commands/install.rs

```diff
@@ -130,34 +133,28 @@ fn extract_archive_into<P: AsRef<Path>>(
     Ok(())
 }
 
-#[cfg(unix)]
 fn package_url(mirror_url: Url, version: &Version) -> Url {
```

Comment:
Instead of duplicating this whole function, I extracted the only part that actually varies into a helper. I also use this helper for the "==> Extrating ..." message.